package.json

```diff
@@ -42,8 +42,10 @@
     "test:py:report-coverage": "$VIRTUAL_ENV/bin/coveralls",
     "test:py:check-coverage": "coverage3 report --fail-under=100",
     "test": "run-s test:js test:py",
+    "packtracker": "packtracker-upload --stats=webpack-stats.json --output-path=dist/prod",
     "build": "webpack --config webpack.dev.js",
-    "prod": "webpack --config webpack.prod.js"
+    "prod": "webpack --config webpack.prod.js",
+    "prod:stats": "webpack --config webpack.prod.js --json > webpack-stats.json"
```

Comment:
Should `webpack-stats.json` be in our `.gitignore`?

Reply:
Sure -- this command is never run locally (only on CircleCI), so shouldn't be included in any commits anyway, but I think that makes sense.

Reply:
I'll give a 👍 once that's in, then.